### PR TITLE
Simplified custom liquid tags and created documentation

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,5 +72,18 @@ the given file into your `.git/hook` directory:
 
     cp ./_devtools/git-hooks/post-commit .git/hooks/
 
+## Custom liquid tags
+
+Custom liquid tags are used to make linking to javadocs, properties, and documents easier
```
Comment (keith-turner) [inline README.md]: Could add a sentence mentioning/linking to source.

File: README.md [hunk 2]
```diff
@@ -72,5 +72,18 @@ the given file into your `.git/hook` directory:
...
+
+| Tag   | Description            | Options                                                                         | Examples                                             | 
+| ----- | ---------------------- | ------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| jlink | Creates Javadoc link   | Link text will be class name by default. Use `-f` for full package + class name | `jlink -f org.apache.accumulo.core.client.Connector` |
```
Comment (keith-turner) [inline README.md]: Why not use `{% ... %}` in examples?

File: _docs-2-0/administration/caching.md [hunk 3]
```diff
@@ -37,13 +37,9 @@ conn.tableOperations().setProperty("mytable", "table.cache.block.enable", "true"
...
 their defaults by setting the following properties:
 
-* [tserver.cache.data.size]
-* [tserver.cache.index.size]
+* {% plink tserver.cache.data.size %}
+* {% plink tserver.cache.index.size %}
 
-[tserver]: {{ page.docs_baseurl }}/getting-started/design#tablet-server-1
-[RFiles]: {{ page.docs_baseurl}}/getting-started/design#rfile
-[table.cache.block.enable]: {{ page.docs_baseurl }}/administration/properties#table_cache_block_enable
```
Comment (keith-turner) [inline _docs-2-0/administration/caching.md]: Nice to see this so easily inlined.

File: README.md (hunk 1 above)
Comment (mikewalch) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (mikewalch) [reply README.md]: Oops.  It would be better to include them. Fixed now

**Pull request merged**